src/server.rs

```diff
@@ -280,22 +280,41 @@ impl Server {
         function: FunctionInfo,
         is_const: bool,
     ) {
+
+        let second_route =  if route.ends_with("/") {
+             &route[0..route.len() - 1]
+        } else {
+             &format!("{}/", route)
+        };
+
+        self._add_route(py, route_type, route, &function, is_const);
+        self._add_route(py, route_type, second_route, &function, is_const);
+    }
+
+    fn _add_route(
+        &self,
+        py: Python,
+        route_type: &HttpMethod,
+        route: &str,
+        function: &FunctionInfo,
+        is_const: bool,
+    ) {
         debug!("Route added for {:?} {} ", route_type, route);
         let asyncio = py.import("asyncio").unwrap();
         let event_loop = asyncio.call_method0("get_event_loop").unwrap();
 
         if is_const {
             match self
                 .const_router
-                .add_route(route_type, route, function, Some(event_loop))
+                .add_route(route_type, route, function.clone(), Some(event_loop))
```

Comment:
Why add clones here?

Reply:
to pass `FunctionInfo` from `&FunctionInfo`, either I should clone here, or before (in `#add_route`). I checked whether we can pass it directly, but we can't we would want to do `#clone` on `Router#add_route`.